cluster/cluster_test.go

```diff
@@ -334,6 +334,7 @@ func testTLSConnection(t *testing.T) {
 	)
 	require.NoError(t, err)
 	go p2.Settle(context.Background(), 0*time.Second)
+	require.NoError(t, p2.WaitReady(context.Background()))
```

Comment:
https://github.com/prometheus/alertmanager/pull/3589 already included this so closing this.

Reply:
Thanks though!